src/ForceTorque_TEST.cc

```diff
@@ -22,6 +22,38 @@
 TEST(DOMForceTorque, Construction)
 {
   sdf::ForceTorque ft;
+  sdf::Noise defaultNoise, noise;
```

Comment:
It would be nice to add an integration test to make sure values are parsed correctly from SDFormat files.

Reply:
Added to the joint sensors dom integration test: e4753c2